### PR TITLE
bpo-45766: Add direct proportion option to linear_regression().

### PR DESCRIPTION
**Pull request by rhettinger**
https://bugs.python.org/issue45766

### CODE REVIEW

File: Doc/library/statistics.rst [hunk 1]
```diff
@@ -677,8 +677,18 @@ However, for reading convenience, most of the examples show sorted sequences.
...
+   The data is fit to a line passing through the origin.
+   Since the *intercept* will always be 0.0, the underlying linear
+   function simplifies to:
+
+      *y = slope \* x + noise*
+
    .. versionadded:: 3.10
 
+   .. versionchanged:: 3.11
+      Added support for *proportional*.
```
Comment (erlend-aasland) [inline Doc/library/statistics.rst:691]: You need to update the argument spec in the docs as well. Line 646.

File: Misc/NEWS.d/next/Library/2021-11-09-09-18-06.bpo-45766.dvbcMf.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Added *proportional* option to statistics.linear_regression().
```
Comment (erlend-aasland) [inline Misc/NEWS.d/next/Library/2021-11-09-09-18-06.bpo-45766.dvbcMf.rst]:
```suggestion
Added *proportional* option to :meth:`statistics.linear_regression`.
```

File: Doc/library/statistics.rst (hunk 1 above)
Comment (rhettinger) [reply Doc/library/statistics.rst:691]: Done.

File: Doc/library/statistics.rst [hunk 3]
```diff
@@ -643,7 +643,7 @@ However, for reading convenience, most of the examples show sorted sequences.
 
    .. versionadded:: 3.10
 
-.. function:: linear_regression(x, y, /)
+.. function:: linear_regression(x, y, /, * proportional=False)
```
Comment (erlend-aasland) [inline Doc/library/statistics.rst]:
```suggestion
.. function:: linear_regression(x, y, /, *, proportional=False)
```

**Review by erlend-aasland — APPROVED** [review]

**Pull request merged**